tools/study-room-scraper/src/index.ts

```diff
@@ -0,0 +1,31 @@
+import { PrismaClient } from "@libs/db";
+import type { StudyLocation, StudyRoom } from "@peterportal-api/types";
+
+import { scrapeStudyLocations } from "./study-room-scraper";
+
+const prisma = new PrismaClient();
+
+async function main() {
+  const studyLocations: { [id: string]: StudyLocation } = await scrapeStudyLocations();
```

Comment:
The function return has already been annotated. You don't need to annotate when consuming the function, unless a type assertion is needed.

```suggestion
  const studyLocations= await scrapeStudyLocations();
```